services/apps/portal/src/application/application.service.ts

```diff
@@ -202,7 +202,7 @@ export class ApplicationService {
           typeCode: application.typeCode,
           dateSubmittedToAlc: Date.now().toString(),
           documents: application?.documents.map((d) => ({
-            type: d.type,
+            type: d.type!, //TODO: Do we verify this?
```

Comment:
I think we do check file type on api level

Reply:
This idea is around checking that each document has a type before we submit to ALCS, we might need some other validation steps as well as we start building out more features.